docs/runbooks/KubeMacPoolDuplicateMacsFound.md

```diff
@@ -1,32 +1,40 @@
 # KubeMacPoolDuplicateMacsFound
+<!--apinnick, Oct. 2022-->
 
 ## Meaning
 
-KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.  
-Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.  
-In case it finds duplicates it will raise this alert.
+This alert fires when `KubeMacPool` detects duplicate MAC addresses.
+
+`KubeMacPool` is responsible for allocating MAC addresses and preventing MAC address conflicts. When `KubeMacPool` starts, it scans the cluster for the MAC addresses of virtual machines (VMs) in managed namespaces.
 
 ## Impact
 
-Duplicate MAC Addresses on the same LAN may cause unexpected and various connectability issues in the network.
+Duplicate MAC addresses on the same VLAN might cause network issues.
```

Comment:
While in theory a network can have duplicate MAC addresses if the VLAN on the NADs is different - KMP does not make that distinction. There is no relevance to VLAN here.
```suggestion
Duplicate MAC addresses on the same LAN might cause network issues.
```

Reply:
Fixed